### DISCUSSION TITLE
The purpose of using duplicate semicolons

### DISCUSSION THREAD

**Discussion by Kalfeu** [PrincetonUniversity/athena#588] [Q&A] [answered]
I have found duplicate semicolons(;;) in `src/outputs/io_wrapper.cpp`, `src/pgen/chem_G14Sod.cpp`, `src/pgen/fft.cpp`, `src/pgen/poisson.cpp` and `src/pgen/binary_gravity.cpp` as shown below. What is the purpose of using duplicate semicolons there?
![image](https://github.com/PrincetonUniversity/athena/assets/164971206/ddaf2ea8-9edf-4a27-ae05-45d7ac1bcb39)
![image](https://github.com/PrincetonUniversity/athena/assets/164971206/9958a606-a0a1-47a7-97ad-c4253bd7dd5e)

**Comment by tomidakn** [comment, marked as answer]
Those are most likely just typos which are not covered by the style checker.

**Reply by Kalfeu** [reply]
Similarly, duplicate commas(,,) appear in `src/mesh/mesh_refinement.cpp` (at line 630):
https://github.com/PrincetonUniversity/athena/blob/f701e6785d710ef6a90fb23254e4932b8de27ff6/src/mesh/mesh_refinement.cpp#L627-L632 Is it a typo?

**Reply by tomidakn** [reply]
I think so (though it is commented out anyway).

The duplicate semicolons do nothing. We should clean them up in the next update.
